stan/math/prim/scal/prob/beta_lccdf.hpp

```diff
@@ -32,14 +32,16 @@ namespace stan {
 namespace math {
 
 /**
- * Returns the beta log complementary cumulative distribution function for
- * the given probability, success, and failure parameters. Given matching
- * containers returns the log sum of probabilities.
+ * Returns the beta log complementary cumulative distribution function
+ * for the given probability, success, and failure parameters.  Any
+ * arguments other than scalars must be containers of the same size.
+ * With non-scalar arguments, the return is the sum of the log ccdfs
+ * with scalars broadcast as necessary.
```

Comment:
I'd prefer "With mixed scalar and container arguments, the scalars are broadcast to containers of length matching the container arguments. With container arguments, this funcion returns the sum of lccdfs."

It makes the broadcast behavior more explicit? As necessary seems vague.

But given this is how stuff is documented everywhere.

I'd be tempted to say just do whatever all the other lccdfs do, but "log sum of probabilities" seems wrong (they're sums of logs of probabilities). Is that why you modified this? Bah, docs are hard.